### PR TITLE
v8

### PR DESCRIPTION
**Pull request by azhawkes**
* Ignore navigations that are automatic or are the side effect of another event.
* Fix a bug in Firefox that caused events in iframes to not be recorded.
* Upgrade the polyfill.

### CODE REVIEW

File: js/windowEventRecorder.js [hunk 1]
```diff
@@ -39,12 +39,10 @@ const recordEvent = (e) => {
...
         if (window.parent !== window) {
             let frame = window
-            let frameElement = window.frameElement
             let frameTag = window.frameElement ? window.frameElement.tagName.toLowerCase() : 'iframe';
 
             if (frame.name) {
                 attrs['frameName'] = frame.name
-                attrs['frameElement'] = frameElement
 
                 selector = `${frameTag}[name="${frame.name}"] ${selector}`
```
Comment (konfrontend) [inline js/windowEventRecorder.js:47]: Wow, cool! This works for me too. Not very clear **why** though

Comment (azhawkes) [reply js/windowEventRecorder.js:47]: I think Firefox just silently refused to send anything in the message (`sendMessage`) that couldn't be JSON-ified. So it was dropping them when an `Element` was included. Took me a while to track it down to that.

**Comment by konfrontend** [comment]
[Initial Navigate step for opening the recording tab is missed](https://trello.com/c/9izx0Oaj/461-initial-navigate-step-for-opening-the-recording-tab-is-missed)

**Comment by azhawkes** [comment]
Forgot to merge this but v8 is already live in the directories. Making a new branch for the fix.

**Pull request merged**